lib/Widget/shared/alert_style.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:AbdoCare_Web/page/dashboard_postHos.dart';
+import 'package:AbdoCare_Web/page/dashboard_pre.dart';
+import 'package:flutter/material.dart';
+
+class Dialogs {
+  static Future<void> alertDialogNoData(BuildContext context) async {
+    await showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return AlertDialog(
+            shape:
+                RoundedRectangleBorder(borderRadius: BorderRadius.circular(15)),
+            title: Text("กรุณาทำแบบประเมินให้ครบถ้วน",
+                style: Theme.of(context).textTheme.bodyText2,
+                textAlign: TextAlign.center),
+            actions: [
+              FlatButton(
+                child: Text('ตกลง'),
+                onPressed: () {
+                  Navigator.pop(context);
+                },
+              )
+            ]);
+      },
+    );
+  }
+
+  static Future<void> alertDialogSaveData(
+      BuildContext context, String hn, String patients) async {
+    await showDialog(
+      context: context,
+      builder: (BuildContext context) {
```

Comment:
alert_SuccessfullySavedData sth like that

Reply:
Thank ka

Reply:
solve in dee4924